hadoop-ozone/dist/src/main/compose/ozones3-haproxy/docker-compose.yaml

```
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "3"
+services:
+   s3g:
+      image: haproxy:latest
+      volumes:
+         - ../..:/opt/hadoop
+         - ./haproxy-conf/haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg
+      ports:
+         - 8081:9878
```

@bharatviswa504, it's just a guess, but can you try the following change?

```suggestion
         - 9878:9878
```

---

This is when aws cli tests are run from outside docker, but we execute these tests inside docker network. So, I think this should not be a problem. But I will give a try, as I was not able to figure out why it is failing in Jenkins.